src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -262,9 +272,87 @@ const ProteinExternalReference = (props: ProteinExternalReferenceProps) => {
 
   return (
     <div className={styles.proteinExternalReference}>
-      <ExternalReference label={props.source} to={url} linkText={props.name} />
+      <ExternalReference
+        label={props.source}
+        to={url}
+        linkText={props.accessionId}
+      />
     </div>
   );
 };
 
+type ProteinExternalReferenceGroupProps = {
+  source: ExternalSource;
+  xrefs: Array<
+    Pick<ExternalReferenceType, 'accession_id' | 'name'> &
+      Pick2<ExternalReferenceType, 'source', 'id'>
+  >;
+};
+
+const ProteinExternalReferenceGroup = (
+  props: ProteinExternalReferenceGroupProps
+) => {
+  const { source, xrefs } = props;
+
+  const [isXrefGroupOpen, setXrefGroupOpen] = useState(false);
+
+  const toggleXrefGroup = () => {
+    setXrefGroupOpen(!isXrefGroupOpen);
+  };
+
+  const chevronClasses = classNames(styles.chevron, {
+    [styles.chevronUp]: isXrefGroupOpen
+  });
+
+  if (xrefs.length > 3) {
+    const displayXref = xrefs[0];
+    return (
+      <>
+        <div className={styles.xrefWithChevron}>
+          <ProteinExternalReference
+            key={displayXref.accession_id}
+            source={source}
+            accessionId={displayXref.accession_id}
+            name={displayXref.name}
+          />
+          <div className={styles.xrefGroupChevron} onClick={toggleXrefGroup}>
+            <span
+              className={isXrefGroupOpen ? styles.xrefCountChevronOpen : ''}
```

Comment:
<img src="https://user-images.githubusercontent.com/6834224/117165402-5f0c5500-adbd-11eb-9001-6c2708091b25.png" width="500px" />

Meh. We don't really need a class attribute without any value. If you default to `undefined` instead of the `''`, the span shouldn't have an empty `class` attribute.